packages/api/src/external/hie/tally-doc-query-progress.ts

```diff
@@ -102,6 +107,7 @@ export function setHIETallyCount(
       ...sourceProgress[type],
       successful: sourceSuccessful + tallySuccessful,
       errors: sourceErrors + tallyErrors,
+      total: sourceTotal,
```

Comment:
Found this missing while unit testing it. If the external data for that source/type is empty, we were not setting the total to zero. This fixes it.